data/mods/Magiclysm/Spells/attunements/Crusader.json

```diff
@@ -0,0 +1,116 @@
+    "max_level": 35,
+    "spell_class": "CRUSADER",
+    "energy_source": "MANA"
+  },
+  {
+    "type": "technique",
+    "id": "RSTRIKE",
+    "name": "Radiant Strike",
+    "melee_allowed": true,
+    "crit_tec": true,
+    "messages": [
+      "Your sword bursts with radiant light, hitting everyone nearby.",
+      "<npcname>'s sword bursts with radiant light, hitting everyone nearby."
+    ],
+    "aoe": "spin",
+    "description": "Attack adjacent enemies, crit only"
+  },
+  {
+    "id": "holy_shield",
+    "type": "SPELL",
+    "name": "Holy Shield",
+    "description": "Summon a radiant shield to protect yourself from attacks. Affords protection from most energy based effects",
+    "valid_targets": [ "self" ],
+    "flags": [ "VERBAL", "CONCENTRATE", "MUST_HAVE_CLASS_TO_LEARN" ],
+    "effect": "spawn_item",
+    "effect_str": "holy_shield",
+    "shape": "blast",
+    "spell_class": "CRUSADER",
+    "energy_source": "MANA",
+    "base_casting_time": 350,
+    "base_energy_cost": 500,
+    "difficulty": 4,
+    "max_level": 35,
+    "min_damage": 1,
+    "max_damage": 1,
+    "min_duration": 360000,
+    "max_duration": 1440000,
+    "duration_increment": 40115
+  },
+  {
+    "id": "holy_shield",
+    "type": "ARMOR",
+    "name": { "str_sp": "Holy Shield" },
+    "description": "A shield of radiant energy worn on your arm. Affords moderate protection from attacks and energy based effects.",
+    "weight": "5 g",
+    "volume": "1 ml",
+    "to_hit": -10,
+    "symbol": "[",
+    "color": "brown",
+    "covers": [ "arm_r", "arm_l" ],
+    "coverage": 100,
+    "encumbrance": 0,
+    "environmental_protection": 2,
+    "relic_data": {
+      "passive_effects": [
+        {
+          "has": "WORN",
+          "condition": "ALWAYS",
+          "values": [
+            { "value": "ARMOR_ACID", "multiply": -0.2 },
+            { "value": "ARMOR_ELEC", "multiply": -0.2 },
+            { "value": "ARMOR_BIO", "multiply": -0.2 },
+            { "value": "ARMOR_HEAT", "multiply": -0.2 },
+            { "value": "ARMOR_COLD", "multiply": -0.2 }
+          ]
+        }
+      ]
+    },
+    "flags": [ "WATERPROOF", "TRADER_AVOID", "SEMITANGIBLE", "ONLY_ONE", "OVERSIZE", "BELTED", "BLOCK_WHILE_WORN", "STURDY" ],
+    "techniques": [ "WBLOCK_3" ]
+  },
+  {
+    "id": "longsword_judgement",
+    "type": "GENERIC",
+    "name": { "str": "Sword of Judgement" },
+    "color": "light_gray",
+    "symbol": "/",
+    "price": 0,
+    "price_postapoc": 0,
+    "description": "A large sword made of glowing, radiant fire. It bathes the area in light and explodes with radiance when striking enemies.",
+    "material": [ "superalloy" ],
+    "flags": [ "UNBREAKABLE_MELEE", "NONCONDUCTIVE", "NO_REPAIR", "NO_SALVAGE", "MAGIC_FOCUS", "LIGHT_80", "TRADER_AVOID" ],
+    "techniques": [ "RSTRIKE" ],
+    "qualities": [ [ "CUT", 1 ], [ "BUTCHER", 8 ] ],
+    "weight": "1250 g",
+    "volume": "2750 ml",
+    "longest_side": "120cm",
+    "bashing": 0,
```

Comment:
It's a big nerf since many martial arts have damage scaling with bash.
Also, probably the sword should support medieval swordmanship

Reply:
I had completely forgotten about Martial Arts. moved some numbers around to be more in line with a tangible sword. Also added support for ninjutsu because it is a sword.

Reply:
i agree with salty panda here, but i would go so far as to say it should support any martial art that utilizes a sword-shaped object. this is an end-game attunement after all, and you're creating a sword for _your_ usage. martial arts that obviously don't support swords are out, though, like the polearm one.

Reply:
It looks like Niten Ichi-Ryu and Fencing both allow weapons similar to the sword that would make sense, so those are an easy shoe-in. Eskrima and Silat, however, both say they use short blades, but Silat allows the use of spears. Does it make sense to add it to those two?

Reply:
It's something I came up with for Mage's staff but consider this:
Use `use_action` to make the sword switch between different modes.
A short blade that can be used with Eskrima, medium blade with pierce damage - rapier, and longer, heavier blade closer to Zweihander
This will only be truly useful once we will have some intra-melee range Kevin's planning, but adds some really nic fluff I think

Reply:
I really like that idea. Tested it and it does keep the timer working properly. Sadly, I can't let the player just select one of the forms, but making the transformations cyclical is easy enough. I think it'd be better to keep the transformation move cost free, though. I think it'd feel bad to need, say, the larger version, but it's the last in the cycle and you've got a zombie in your face.